siliconcompiler/schema/schema_cfg.py

```diff
@@ -759,31 +759,40 @@ def schema_datasheet(cfg, design='default', name='default', mode='default'):
             standard footprint names or a custom naming methodology used in
             conjunction with 'fileset' names in the output parameter.""")
 
-    # Absolute max voltage
-    scparam(cfg, ['datasheet', design, 'limits', 'voltage', name],
-            sctype='(float,float)',
-            shorthelp="Datasheet: absolute voltage limits",
-            switch="-datasheet_limits_voltage 'design pin <(float,float)>'",
-            example=[
-                "cli: -datasheet_limits_voltage 'mydevice vdd (-0.4,1.1)'",
-                "api: chip.set('datasheet','mydevice','limits','voltage','vdd', (-0.4,1.1))"],
-            schelp="""Device absolute minimum/maximum voltage not to be
-            exceeded, specified on a per pin basis.""")
-
     # Absolute max temperatures
     metrics = {'storagetemp': 'storage temperature limits',
+               'soldertemp': 'solder temperature limits',
                'junctiontemp': 'junction temperature limits'}
 
     for item, val in metrics.items():
         scparam(cfg, ['datasheet', design, 'limits', item],
                 sctype='(float,float)',
                 shorthelp=f"Datasheet: absolute {val}",
-                switch=f"-datasheet_{item} 'design <(float,float)>'",
+                switch=f"-datasheet_limits_{item} 'design <(float,float)>'",
                 example=[
-                    f"cli: -datasheet_{item} 'mydevice (-40,125)'",
+                    f"cli: -datasheet_limits_{item} 'mydevice (-40,125)'",
                     f"api: chip.set('datasheet','mydevice','limits','{item}',(-40,125))"],
                 schelp=f"""Device absolute {val} not to be exceeded.""")
```

Comment:
Add tuple of (min, max)

Reply:
It is a tuple. Can you clarify?

Reply:
Sorry that's not clear, add to the help that this is a (min, max). Like the min, typ, max ones above